### DISCUSSION TITLE
What is the best way to handle multilanguage ?

### DISCUSSION THREAD

**Discussion by t-prod** [filamentphp/filament#12450] [Help]
### Package

Form builder

### Package Version

v3.2.0

### How can we help you?

Hi,

What is the best way to manage multilanguage ?

The idea is to have a language switcher with the locales we would like.
And have the ability to setLocale() and have fields with flags

I see different plugin for multilanguage.

What do you think ?

**Comment by leandrocfe** [comment]
Did you try Spatie Translatable? This is the official plugin

https://filamentphp.com/plugins/filament-spatie-translatable

**Comment by t-prod** [comment]
Hi,

Yep but I'm using all Filament packages in a custom admin without Panel package.
This plugin is OK to translate fields
Do you know how can I generate the lang switcher by using his plugin ?
Do you think it is possible ?
